beacon-chain/node/node.go

```diff
@@ -819,3 +825,31 @@ func (b *BeaconNode) registerInteropServices() error {
 	}
 	return nil
 }
+
+func (b *BeaconNode) registerValidatorMonitorService() error {
```

Comment:
Might as well use the new init pattern adopted by @rauljordan 

https://github.com/prysmaticlabs/prysm/pull/9783